bytecomp/debuginfo.ml

```diff
@@ -54,3 +54,17 @@ let from_location kind loc =
 
 let from_call ev = from_location Dinfo_call ev.Lambda.lev_loc
 let from_raise ev = from_location Dinfo_raise ev.Lambda.lev_loc
+
+let to_location d =
+  if is_none d then Location.none
+  else
+    let loc_start =
+      { Lexing.
+        pos_fname = d.dinfo_file;
+        pos_lnum = d.dinfo_line;
+        pos_bol = 0;
+        pos_cnum = d.dinfo_char_start;
```

Comment:
This does not seem to follow the usual semantics of Lexing.location, where cnum (resp. bol) is the number of characters from the beginning of the file (resp. line).  This might be fine considering how the info is used in practice for printing purposes, but it seems weird.